airflow/utils/task_group.py

```diff
@@ -48,6 +49,13 @@ class TaskGroup(TaskMixin):
     :type parent_group: TaskGroup
     :param dag: The DAG that this TaskGroup belongs to.
     :type dag: airflow.models.DAG
+    :param default_args: A dictionary of default parameters to be used
+        as constructor keyword parameters when initialising operators.
+        Note that operators have the same hook, and precede those defined
```

Comment:
Can we also add a note here that this will overwrite the `default_args` defined in the DAG level?

Reply:
thanks for suggestion, added it